dags/blocksec_plugin/contract_interaction_operator.py

```diff
@@ -3,6 +3,7 @@
 from blocksec_plugin.web3_hook import Web3Hook
 from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
 from web3.exceptions import InvalidAddress
+from constants.constants import PriceConstants
```

Comment:
This is a great idea, maybe we can refactor all the constants, `AddressConstants` for example

Reply:
but don't do that in this PR, this is just a comment for later